src/bluehawk/parser/lexer/lexer.test.ts

```diff
@@ -116,9 +144,13 @@ describe("custom comment lexer", () => {
 
   it("rejects comment patterns that conflict with other tokens", () => {
     expect(() => {
-      makeLexer([makeLineCommentToken(TAG_PATTERN)]);
```

Comment:
Note for reviewer: this test was failing independent of my changes. I've updated it to try/catch and match on the error message, and now it's passing.